crates/red_knot_python_semantic/resources/mdtest/imports.md

```diff
@@ -0,0 +1,133 @@
+# Follow imports
+
+## Classes
+
+We can follow import to class:
+
+````markdown
+```py path=a.py
```

Comment:
I would say it's probably better style in general if we don't specify the path on the "main" test file, if it isn't imported by another file and its location/name don't actually matter to the test. (This wasn't possible in the previous test format.) I don't care too much about this, though, not a blocking issue.

Reply:
Fixed it